sdk/core/core-arm/package.json

```diff
@@ -105,8 +105,9 @@
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "tslint -p . --fix -c tslint.json --exclude test/**/*.ts",
-    "lint": "tslint -p . -c tslint.json --exclude test/**/*.ts",
+    "lint:fix": "eslint -c ../../.eslintrc.json src tests samples --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint -c ../../.eslintrc.json src tests samples --ext .ts -f node_modules/eslint-detailed-reporter/lib/detailed.js -o core-arm-lintReport.html || exit 0",
+    "lint:terminal": "eslint -c ../../.eslintrc.json src tests samples --ext .ts",
```

Comment:
Does the regular “lint” task not provide terminal output of the lint issues? I thought it did both (report on the terminal AND write to the HTML file).

Reply:
Oh no it doesn't do both. It's like output to terminal by default and to the HTML file with the flag -f

Reply:
Looks like (of course) there is a package we could use for this: https://github.com/lwhiteley/eslint-output

Feel like that would be a valuable change - maybe not in this PR though. But we already do the same thing for Mocha for example: we display results to the console and generate the junit xml report.

Reply:
I can open an issue to track this

Reply:
Sounds good!